python/remotivelabs-broker/remotivelabs/broker/sync/helper.py

```diff
@@ -39,7 +39,7 @@ def intercept(
         return method(request_or_iterator, new_details)
 
 
-def create_channel(url: str, x_api_key: str = 'offline'):
+def create_channel(url: str, x_api_key: Optional[str] = None):
```

Comment:
I like! But does it have impact on usage? In that case are samples updated?

Reply:
Won't affect our samples is practice; they will still execute. But the samples should use `None` as default instead of `'offline'` which they use at the moment.

I'll update samples after merging this.